### PR TITLE
fix: pnpm clean in windows

### PR DESCRIPTION
**Pull request by arpansaha13**
## :bookmark_tabs: Summary

pnpm install is not working in windows.

Resolves #<your issue id here>

## :straight_ruler: Design Decisions

Describe the way your implementation works or what design decisions you made if applicable.

### :clipboard: Tasks

Make sure you

- [x] :book: have read the [contribution guidelines](https://github.com/mermaid-js/mermaid/blob/develop/CONTRIBUTING.md)
- [ ] :computer: have added unit/e2e tests (if appropriate)
- [x] :bookmark: targeted `develop` branch

### CODE REVIEW

**Comment by sidharthv96** [comment]
Can you provide more info?
Which OS were you using?

I'm able to run the current pie demo in develop without any issues. 
The `import` statement is actually deliberately added to showcase the esm import feature.

**Comment by arpansaha13** [comment]
Oh, I didn't know that the import was intentional. I am using Windows 10.

Actually when I run `pnpm i` after a fresh clone, I got errors about the `clean` npm-script. I think the problem is with the semi-colon syntax in npm-script. I read in an article that semi-colon syntax does not work in Windows.

If I replace the semi-colon with `&&`, then the pie-demo works again, with the same import.

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "build:vite": "ts-node-esm --transpileOnly --project=.vite/tsconfig.json .vite/build.ts",
     "build:types": "concurrently \"tsc -p ./packages/mermaid/tsconfig.json --emitDeclarationOnly\" \"tsc -p ./packages/mermaid-mindmap/tsconfig.json --emitDeclarationOnly\"",
     "build:watch": "pnpm build:vite --watch",
-    "build": "pnpm clean; concurrently \"pnpm build:vite\" \"pnpm build:types\"",
+    "build": "(pnpm clean || true) && concurrently \"pnpm build:vite\" \"pnpm build:types\"",
```
Comment (sidharthv96) [inline package.json:33]: || true is not the right workaround. 

can you try renaming clean to cleanup here and line 29?

Comment (arpansaha13) [reply package.json:33]: I've replaced with cleanup and it still doesn't work.

image

image

I've also tried with a random name:

image

image

But they work if semi-colon is replaced:

image

image

Comment (sidharthv96) [reply package.json:33]: You can actually remove clean from the root package.json.

Try if `pnpm run -r clean` works inside build?

Comment (arpansaha13) [reply package.json:33]: I have removed clean from root, and added the command you gave.

This was the first run:

image

Then I put a space between clean and ; and this is second run:

image

Here the second part of the script is not running.

Comment (sidharthv96) [reply package.json:33]: You can delete `;` and use &&.

**Comment by sidharthv96** [comment]
@arpansaha13 Did it work?

**Comment by arpansaha13** [comment]
yes it worked :)

image

**Comment by sidharthv96** [comment]
Thanks for the contribution @arpansaha13 :)

**Pull request merged**